### ISSUE TITLE
I didn't find the "rowGap" and "gridRowGap" properties in the 'dart: html' for an HtmlElement style or for getComputedStyle ()

### ISSUE THREAD

**Issue by insinfo** [dart-lang/sdk#46077]
I didn't find the "rowGap" and "gridRowGap" properties in the 'dart: html' for an HtmlElement style or for getComputedStyle ()

I'm using AngularDart 6 with dart 2.12.4

```dart
 var gridsElemets = <html.HtmlElement>[...html.document.querySelectorAll('.grid--masonry')];
    
    var grids;
    if (gridsElemets.isNotEmpty && gridsElemets[0].getComputedStyle().gridTemplateRows != 'masonry') {
      grids = gridsElemets.map((grid) => {
            '_el': grid,
             'gap': double.tryParse(grid.getComputedStyle().gridRowGap),
             'items': <html.HtmlElement>[...grid.childNodes ].where((c) => c.nodeType == 1 && c.getComputedStyle().gridColumnEnd != '-1'),
            'ncol': 0
          });
    }
```
I am trying to translate this JavaScript code to dart
```js
let grids = [...document.querySelectorAll('.grid--masonry')];

if (grids.length && getComputedStyle(grids[0]).gridTemplateRows !== 'masonry') {
  grids = grids.map(grid => ({
    _el: grid,
    gap: parseFloat(getComputedStyle(grid).gridRowGap),
    items: [...grid.childNodes].filter(c => c.nodeType === 1 && +getComputedStyle(c).gridColumnEnd !== -1),
    ncol: 0 }));


  function layout() {
    grids.forEach(grid => {
      /* get the post relayout number of columns */
      let ncol = getComputedStyle(grid._el).gridTemplateColumns.split(' ').length;

      /* if the number of columns has changed */
      if (grid.ncol !== ncol) {
        /* update number of columns */
        grid.ncol = ncol;

        /* revert to initial positioning, no margin */
        grid.items.forEach(c => c.style.removeProperty('margin-top'));

        /* if we have more than one column */
        if (grid.ncol > 1) {
          grid.items.slice(ncol).forEach((c, i) => {
            let prev_fin = grid.items[i].getBoundingClientRect().bottom /* bottom edge of item above */,
            curr_ini = c.getBoundingClientRect().top /* top edge of current item */;

            c.style.marginTop = `${prev_fin + grid.gap - curr_ini}px`;
          });
        }
      }
    });
  }

  addEventListener('load', e => {
    layout(); /* initial load */
    addEventListener('resize', layout, false); /* on resize */
  }, false);
}
```
I'm trying to implement this code
https://css-tricks.com/a-lightweight-masonry-solution/

https://www.w3schools.com/cssref/css3_pr_row-gap.asp
https://www.w3schools.com/cssref/pr_grid-row-gap.asp
https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap

**Comment by insinfo** [comment]
I just realized that the 'backdrop-filter' property is also missing

**Comment by rileyporter** [comment]
Added `gap` and `row-gap` in [this recent CL](https://github.com/dart-lang/sdk/commit/fc8a65d4c8bed75a1f94889b4d04ac61dc38c20d). The `column-gap` property was already there from WebKit. We decided to not add the deprecated `grid-row-gap` or `grid-column-gap` names since the suggestion is to use the updated `row-gap` and `column-gap` names.

I filed a separate bug, #47996, for adding the `backdrop-filter` property. It's not supported in Firefox, so we want to think a little bit about the best way to add it to dart:html. Workaround using js_util to access the property included in that bug.
